hyrax.gemspec

```diff
@@ -79,4 +79,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'shoulda-matchers', '~> 3.1'
   spec.add_development_dependency 'rails-controller-testing', '~> 0'
   spec.add_development_dependency 'webmock'
+  spec.add_development_dependency 'i18n-debug' unless ENV['TRAVIS'] == 'true'
```

Comment:
Because `spec_helper.rb` was itself internally inconsistent. 😆  Fixed that in my follow-up. Thx.